pkg/operator/certrotation/cabundle.go

```diff
@@ -88,14 +89,16 @@ func (c CABundleConfigMap) EnsureConfigMapCABundle(ctx context.Context, signingC
 	}
 
 	if modified {
-
-		actualCABundleConfigMap, updated, err := resourceapply.ApplyConfigMap(ctx, c.Client, c.EventRecorder, caBundleConfigMap)
+		var actualCABundleConfigMap *corev1.ConfigMap
+		if !exists {
+			actualCABundleConfigMap, err = c.Client.ConfigMaps(c.Namespace).Create(ctx, caBundleConfigMap, metav1.CreateOptions{})
+		} else {
+			actualCABundleConfigMap, err = c.Client.ConfigMaps(c.Namespace).Update(ctx, caBundleConfigMap, metav1.UpdateOptions{})
```

Comment:
why don't want to report https://github.com/openshift/library-go/blob/master/pkg/operator/resource/resourceapply/event_helpers.go#L34 anymore ?

Reply:
Good point, we get an event that its about to be updated, but not the update result. Added this to the latest commit